app/views/content/how-to-apply-for-teacher-training/if-your-application-is-unsuccessful.md

```diff
@@ -51,6 +51,8 @@ It’s worth looking at the feedback the course provider has given to understand
 
 There could be a number of reasons your application was unsuccessful:
 
+$visa-sponsorship$
```

Comment:
@emmaachesongray I'm not sure this makes sense to have this as an expander - and particularly not this high up the page

I don't think this is better than what is already in production! Let's discuss